tests/e2e/testsLibrary/LsTests.ts

```diff
@@ -8,81 +8,88 @@
  * SPDX-License-Identifier: EPL-2.0
  **********************************************************************/
 
-import { e2eContainer } from '../inversify.config';
-import { Editor, CLASSES, Ide, TimeoutConstants } from '..';
+import 'reflect-metadata';
+import { inject, injectable } from 'inversify';
+import { CLASSES } from '../inversify.types';
+import { TimeoutConstants } from '../TimeoutConstants';
+import { Editor } from '../pageobjects/ide/Editor';
+import { Ide} from '../pageobjects/ide/Ide';
 import { Key, error } from 'selenium-webdriver';
 import { Logger } from '../utils/Logger';
 
-const editor: Editor = e2eContainer.get(CLASSES.Editor);
-const ide: Ide = e2eContainer.get(CLASSES.Ide);
+@injectable()
+export class LsTests {
```

Comment:
I would prefer to use the full name without abbreviations where possible.
Abbreviations may be tricky, especially for third persons.
What do you think about _**LanguageServerTests**_ ?

Reply:
sounds good, I'll fix that